friends/src/App.js

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from "react";
+import axios from "axios";
+import { Route, Redirect } from "react-router-dom";
+import { FriendsList } from "./components/FriendsList";
+import { NewFriendForm } from "./components/NewFriendForm";
+import uuid from "uuid";
+import "./App.css";
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      friends: [],
+      potentialNewFriend: {
+        id: "",
+        name: "",
+        age: "",
+        email: "",
+        color: "",
+        favefood: "",
+        quotation: ""
```

Comment:
something that could save you time later when needing to reset you field input values after a submission/unwanted action: declare potentialNewFriend outside the component itself up above as `initialNewFriend` or something exactly as you do here, and then in your CDM set `potentialNewFriend: initialNewFriend` to get them all. then when you need to reset all the fields, just update potentialNewFriend (that's wordy to type multiple times too), set it back to `initialNewFriend` to save yourself time of having to write all those blank strings again